DEMO/org.openl.rules.demo/jetty-conf/start.cmd

```diff
@@ -0,0 +1,10 @@
+@setlocal
+@SET JETTY_HOME_WITH_END_SLASH=%~dp0
+@SET "JETTY_HOME=%JETTY_HOME_WITH_END_SLASH:~0,-1%"
+@IF "%JAVA_HOME%"=="" (
+@    echo JAVA_HOME is not set, cannot run Jetty
+@    exit /B 3
+)
+
+@%JAVA_HOME%\bin\java.exe -DDEMO=DEMO -Dopenl.home=%JETTY_HOME%\openl-demo   -Djava.security.manager -Djava.security.policy=demo-java.policy -Djetty.home=%JETTY_HOME% -Djetty.base=%JETTY_HOME% -Djava.io.tmpdir=%TEMP% -jar %JETTY_HOME%\start.jar jetty.state=%JETTY_HOME%\jetty.state jetty-started.xml
+@endlocal
```

Comment:
As far as I remember, previously we had "smart" RAM allocation, and enabling extra modules if Java version 9+
Could you keep it?